start/config/src/main/java/org/ethereum/beacon/emulator/config/main/ValidatorKeys.java

```diff
@@ -0,0 +1,68 @@
+package org.ethereum.beacon.emulator.config.main;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.util.List;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME)
+@JsonSubTypes({
+    @JsonSubTypes.Type(value = ValidatorKeys.Generate.class, name = "generate"),
+    @JsonSubTypes.Type(value = ValidatorKeys.Private.class, name = "private"),
+    @JsonSubTypes.Type(value = ValidatorKeys.Public.class, name = "public"),
+})
+public abstract class ValidatorKeys {
+
+  public static class Generate extends ValidatorKeys {
+    private int count;
```

Comment:
It would be great to see something like 10 per epoch or, say, one per every 10 slots.

Reply:
Not sure what do you mean...